### ISSUE TITLE
[Web/Wasm] TextField not opening keyboard in mobile browser

### ISSUE THREAD

**Issue by Not-Soti** [JetBrains/compose-multiplatform#4836]
**Describe the bug**
Given a Compose Web app with a TextField, if it is accessed through a browser using a mobile phone, clicking on a text field does not open the software keyboard.
The TextField is receiving the focus as the labelmoves to the top side.

**Affected platforms**

- Web (K/Wasm) - Canvas based API - Accesing with a mobile device
- 
**Versions**
- Compose Multiplatform version: 1.6.6
- Kotlin version: 1.9.23
- Device (model or simulator for iOS issues): Nothing Phone - Google Chrome App

**To Reproduce**

1. Download a new project using the Compose Multiplatform Wizard
2. Set the App() content to this:
```
@Composable
fun App() {
    MaterialTheme {
        var input by remember { mutableStateOf("") }
        Box(
            Modifier.fillMaxSize(),
            contentAlignment = Alignment.Center
        ){
            TextField(
                modifier = Modifier,
                value = input,
                label = { Text("Label") },
                onValueChange = { input = it }
            )
        }
    }
}
```
3. Run the "wasmJsBrowserRun" gradle task, or other related local deployment task.
4. Enter the web through a browser using a smart phone
5. Click on the TextField

**Expected behavior**
The software keyboard should be displayed.

**Screenshots**
![image](https://github.com/JetBrains/compose-multiplatform/assets/64324229/d22e6f6b-482f-4965-8b2a-cef8f6d6faf9)

**Additional context**

**Comment by kasem-sm** [comment]
I'm facing this as well on iOS, CMP version - 1.6.10-rc03 and rc01

**Comment by eymar** [comment]
It was implemented in 1.6.10. Please give it a try.
